packages/common/src/store/ui/toast/types.ts

```diff
@@ -8,7 +8,7 @@ export type Toast = {
   content: ToastContent
   type?: ToastType
   key: string
-  timeout?: number
+  timeout?: number | 'MANUAL'
```

Comment:
An empty timeout is set to default in a few places and I didn't want to change that in case there were places that were relying on that. I agree that having this be either a string or number isn't the best, but it felt ok bc the string is only used in the special case. Open to discussion here